draft-ietf-quic-http.md

```diff
@@ -1916,13 +1916,17 @@ Error codes need to be defined for HTTP/2 and HTTP/3 separately.  See
 ## Since draft-ietf-quic-http-16
 
 - Rename "HTTP/QUIC" to "HTTP/3" (#1973)
-- Define INITIAL_PRIORITY frame (#1865, #2075)
-- Remove exclusive reprioritization (#2075)
+- Changes to PRIORITY frame (#1865, #2075)
+  - Permitted as first frame of request streams
+  - Remove exclusive reprioritization
+  - Changes to Prioritized Element Type bits
 - Define DUPLICATE_PUSH frame to refer to another PUSH_PROMISE (#2072)
 - Set defaults for settings, allow request before receiving SETTINGS (#1809,
   #1846, #2038)
 - Clarify message processing rules for streams that aren't closed (#1972, #2003)
-- Removed reservation of error code 0 (#1922)
+- Removed reservation of error code 0 and moved HTTP_NO_ERROR to this value
+  (#1922)
+- Removed prohibition of zero-length DATA frames (#2098)
```

Comment:
The change in PR #2098 was reverted.

Reply:
Good catch!

Reply:
I reverted the change in definition of zero, but I kept the un-prohibition of sending zero-length frames.  There seemed to be clear agreement that this was a preference, and there's no technical reason to prohibit it.

Reply:
Oh.  Sorry for the noise.

Reply:
Ah, right. Thanks for reverting my change @MikeBishop :-)

Reply:
See, I knew that I'd checked that :)